e-vent.ino

```diff
@@ -267,6 +268,13 @@ void loop() {
         displ.writePEEP(round(pressureReader.peep()));
         displ.writePlateauP(round(pressureReader.plateau()));
         setState(IN_STATE);
+
+        // An indicator that shows patient-triggered or time-triggered cycles
+        if(patientTriggered) {
+          displ.showPatientIcon(0,19);
+        } else {
+          displ.showTimeIcon(0,19);
```

Comment:
same

Reply:
Revised by moving the location details to the display